### PR TITLE
.coafile: Add IndentationBear, indent style.css

### LINKED ISSUE #97: coafile: Create coafile and add indentation Bear

### PR DESCRIPTION
**Pull request by kaustubhhiware**
.coafile: Add IndentationBear, indent style.css

As pointed out in the gitter channel, the formatting in style.css
was  mixed up.
This PR corrects my previous unintended commit. 

Closes #97 

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [X] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [X] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by hemangsk** [comment]
LGTM, Two separate commit for each file will be better I think, whatsay?

**Comment by kaustubhhiware** [comment]
@hemangsk , Sure. I thought I would've been asked to squash the commits, so I made a single commit  :sweat_smile: . Do I split the changes into 2 commits then ?

**Comment by hemangsk** [comment]
Yeah, primarily the concept is that one commit should have only one atomic/nuclear change. So we can have two about adding the indentation bear and about correcting indentation.

**Comment by hemangsk** [comment]
Hey Kaustubh, there's this tiny inconsistency in the file [on this line](https://github.com/kaustubhhiware/landing-frontend/blob/b824940cb88a0463e6c0090ee0ed644f9d379dd4/resources/css/style.css#L272 ), 
do you mind fixing that as well, because then it will be like a perfect 💯  % indented. 😃  😉

**Comment by kaustubhhiware** [comment]
@hemangsk All the suggested changes have been made. Line 272 has been corrected.

2 separate commits have been made to make the changes atomic. Open for review.

File: resources/css/style.css [hunk 1]
```diff
@@ -649,15 +649,15 @@ a:hover > .card{
...
-    padding-right: 1px;
-    float: left;
-    width: 33.33333333%;
+  position: relative;
+  min-height: 1px;
+  padding-left: 1px;
+  padding-right: 1px;
+  float: left;
+  width: 33.33333333%;
 }
```
Comment (hemangsk) [inline resources/css/style.css]: The whitespace is unneeded here, right?

File: resources/css/style.css [hunk 2]
```diff
@@ -649,15 +649,15 @@ a:hover > .card{
...
```
Comment (hemangsk) [inline resources/css/style.css]: Extra newline

**Comment by hemangsk** [comment]
Awesome! just two minor stuff and we're all set 🚀  🚀

**Comment by kaustubhhiware** [comment]
The changes have been made. Please see.

**Comment by hemangsk** [comment]
ack e9e07bf 92bc2a6

**Comment by hemangsk** [comment]
@rultor merge!

**Pull request merged**